POUF1.md

```diff
@@ -0,0 +1,442 @@
+
+To access a reliable current time as needed for the [In Vehicle Implementation Requirements](https://uptane.github.io/uptane-standard/uptane-standard.html#in-vehicle-implementation-requirements), ECUs in this POUF use a time server that produces a time attestation for each update cycle. The Primary ECU sends the time server nonce tokens from each ECU, then the time server returns a time attestation that contains a signed the current time with all the tokens, as described in the [Deployment Considerations](https://uptane.github.io/deployment/introduction.html). The ASN.1 details of the time server are listed in {Formats}. The time server is used by the reference implementation as a self contained way to ensure secure access to the current time.
+
+Additionally, this POUF supports custom delegated targets roles and terminating delegations, features described as optional in the [Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#targets_role_delegations). As explained in the Standard, terminating delegations are not used by default. To use custom roles, the Targets role on the Image repository sets up a delegation. Delegations used in this POUF must support multi-role delegations, as explained in [TAP 3](https://github.com/theupdateframework/taps/blob/d0818e580c322815a473520f2e8cc5f5eb8df499/tap3.md). In addition, delegation filenames are listed using an enumerated list that supports [wildcards](https://uptane.github.io/uptane-standard/uptane-standard.html#delegations_meta). Delegations are supported in the reference implementation to demonstrate how they can be used to establish a chain of trust for specific images.
+
+Public key identifiers use the hash of the key value, encryption algorithm, and verification scheme for a condensed identifier. The format of these identifiers if further described in {Formats}. This POUF supports rsa and ed25519 hashes for public keys. The hashing algorithm must be specified in the signature method.
+
+This POUF supports asymmetric keys and uses the [same keys for encrypting and signing](https://uptane.github.io/uptane-standard/uptane-standard.html#build-time-prerequisite-requirements-for-ecus). Using the same keys allows for a simplified reference implementation and less key management. In particular, each ECU has a private key and registers the associated public key with the director repository. The ECU will use the private key to sign messages and the director repository may use the public key to encrypt images per ECU. In addition, each metadata role has either online or offline private keys that are used on the director and image repositories. The public keys associated with these roles are used by the ECUs during validation.
```

Comment:
"use the private key to sign messages"  Is this just the manifest or more?

Reply:
Yes, this is just the manifest, I changed it to make that more clear.